loader.py

```diff
@@ -52,6 +53,7 @@ def parse_arguments(args = None):
     parser.add_argument('--split-transactions', help='Creates a separate transaction for each file',
                         action='store_true')
     parser.add_argument('--upload-log-dir', help='Upload destination dir for log file,  if dir in s3, use the format, s3://[bucket]/[prefix]')
+    parser.add_argument('--database-type', help='The database type, can be either neo4j or memgraph')
```

Comment:
Please add "choices" argument, so invalid database type can be caught earlier. See "--mode" for example.